### ISSUE TITLE
UI only shows a manifest textfield instead of the expected dialogs

### ISSUE THREAD

**Issue by christianmue1980a** [spinnaker/spinnaker#3062]
### Issue Summary:

When I want to create a new firewall/load balancer/server group, only a textfield is shown to enter a manifest manually. 

![image](https://user-images.githubusercontent.com/36344241/42907249-832c06b6-8add-11e8-97cc-4f012b1b4cb9.png)

From the documentation (https://www.spinnaker.io/guides/tutorials/codelabs/kubernetes-source-to-prod/), I expected a more sophisticated UI like this

![image](https://user-images.githubusercontent.com/36344241/42907263-963a9038-8add-11e8-8051-2b8f37ccf224.png)

Is this a bug or am I doing something completely wrong?

### Cloud Provider(s):

kubernetes (manifest)

### Environment:

on premise, deployed on k8s

### Feature Area: 

deck

### Additional Details:

I tested with these two deck versions:
gcr.io/spinnaker-marketplace/deck:2.3.1-20180709044906
gcr.io/spinnaker-marketplace/deck:2.2.1-20180507112917

**Comment by sbwsg** [comment]
It appears you've setup a Kubernetes v2 account in Spinnaker but the codelab you're following is for the v1 version of the provider.

There's an equivalent codelab for Kubernetes v2 here: https://www.spinnaker.io/guides/tutorials/codelabs/kubernetes-v2-source-to-prod/

**Comment by christianmue1980a** [comment]
Ah, good point. Thanks! Unfortunately it seems like using both providers in parallel is not very well supported (using v1 when having v2 installed as well caused bugs in the frontend). But I will create a separate issue for that when I follow this approach.
